### PR TITLE
Did some cleanup and refactoring.

### PR DESCRIPTION
**Pull request by Carlson-Jonathan**
Hi Wesley! I am creating my own pull request from my repository into yours. This is how other developers contribute to projects created by other people. Cool hu!

I didn't break any of your code, I just did some clean up and refactoring. Any time you make a pull request, you should always be as descriptive as possible for the person reviewing it. See the buttons for editing text? You can do things like:

### Create Heading Text

... and add quotes from previous conversations:

>I didn't break any of your code, I just did some clean up and refactoring. Any time you make a pull request, you should always be as descriptive as possible 

... and make links:

[https://github.com/WesleyCarlson3712](https://github.com/WesleyCarlson3712)

... and:

    `cout << "Format stuff so it looks like you are copying in actual code!` << endl;
---
Anyway, I would like you to review the changes I made to your code and merge them into your development branch.

### CODE REVIEW

File: Calculator.cpp [hunk 1]
```diff
@@ -1,3 +1,10 @@
+/***************************************************************************************************
+* Description:
+    This is a calculator program. The purpose is to learn how to use 'cin' to get a user input and
+    learn how to code defensively so invalid user inputs cannot crash the code.
+* Author:
+*   Wesley
+***************************************************************************************************/
 #include <iostream>
```
Comment (Carlson-Jonathan) [inline Calculator.cpp:8]: Check this out - if you have questions about the code, you can write comments here to the person who wrote it and have conversation directly on top of the code. If you want them to make changes before you merge, this is the place to do it.

File: find2ndchar.cpp [hunk 2]
```diff
@@ -1,25 +1,22 @@
...
+***************************************************************************************************/
 #include <iostream>
 #include <vector>
 using namespace std;
 
 int thingie(char letter, string word) {
     int number = 0;
     for(int i = 0; i < word.length(); i++) {
-        if(word[i] == letter){
+        if(word[i] == letter) {
```
Comment (Carlson-Jonathan) [inline find2ndchar.cpp:14]: There should be a space between the statement parameters and the first curly brace!

**Review by Carlson-Jonathan — COMMENTED** [review]
I did a review of my own code.

File: Calculator.cpp (hunk 1 above)
Comment (WesleyCarlson3712) [reply Calculator.cpp:8]: hi

**Pull request merged**